src/components/pages/callouts/CalloutIntroPanel.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <CalloutSidePanel :show="show" @close="$emit('close')">
+    <div class="lg:flex lg:flex-col lg:justify-center lg:items-start h-full">
```

Comment:
Sorry I didn't see this before but I think it might be better to apply `flex flex-col items-start` to all breakpoints (i.e. remove `lg:`). It probably doesn't matter in this case but flex affects things like collapsing margins so it's good practice to apply to non-conditionally where possible (or wrap the inner elements in a div so they aren't treated as flex children)